src/test/java/com/ays/admin_user/controller/AdminUserAuthControllerTest.java

```diff
@@ -31,7 +31,7 @@ class AdminUserAuthControllerTest extends AbstractRestControllerTest {
     private AdminUserRegisterService adminUserRegisterService;
 
 
-    private static final AysTokenToAysTokenResponseMapper AYS_TOKEN_TO_AYS_TOKEN_RESPONSE_MAPPER = AysTokenToAysTokenResponseMapper.initialize();
+    private final AysTokenToAysTokenResponseMapper AYS_TOKEN_TO_AYS_TOKEN_RESPONSE_MAPPER = AysTokenToAysTokenResponseMapper.initialize();
```

Comment:
This object should be declared as static.

Reply:
I just added `static `keyword to `AysTokenToAysTokenResponseMapper `. Can we look through it again?

Reply:
This object should be declared as static.

Reply:
I revised the code line.